### ISSUE TITLE
Accessibility: MAS36: Keyboard: "Dismiss this update" Button is not accessible with keyboard.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#39]
#A11yMAS
#MAS36
#Wipro
#vscode-website
#FireFox

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0
Firefox Version:50.1.0

**Repro Steps:**
1.Open [https://code.visualstudio.com/docs ](https://code.visualstudio.com/docs )in edge.
2.Navigate to the "Version1.8" below the header of the page.
3.Try to access the close button [refer screenshot ]

**Actual Result:**
Unable to access the "Dismiss this update" button beside the "Version1.8"  below the header of the page while navigating with keyboard.

**Expected Result:**
The user should be able  to access the all the interactive elements available in the page.

**User Impact:**
Users with visual and/or motor impairments who rely on keyboard may find it difficult while navigating with the tab to use the Application. 

**Screenshots:**
![dismiss this update button not accessible](https://cloud.githubusercontent.com/assets/24825441/21712960/79af633a-d41d-11e6-94eb-41bb8668e33a.png)

**Comment by chrisdias** [comment]
tried setting aria-hidden="false" but that didnt do anything.

**Comment by waderyan** [comment]
fixed with tab-index
